test/workers/web_hook_worker_test.rb

```diff
@@ -34,7 +34,7 @@ class WebHookWorkerTest < ActiveSupport::TestCase
       with(body: web_hook.message,
         headers: {
           'Content-Type' => 'application/json; charset=utf-8',
-          'Host'=>'localhost:80'
+          'Host'=>'localhost'
```

Comment:
Surrounding space missing for operator '=>'.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.